swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+# ***** added for test_wave_05 *****
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "Clothing", condition = 0,age=0):
+        self.category = category
+        self.condition = condition
+        self.age=age
+    
```

Comment:
This import statement is clean and readable 🎉 Great job with this import and inheritance